conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosNodeFetchers.scala

```diff
@@ -502,15 +502,15 @@ private[tezos] trait TezosBlocksDataFetchers {
   }
 
   /** a fetcher of baker rolls for blocks */
-  implicit val bakersFetcherByHash = new FutureFetcher {
+  implicit val bakersFetcher = new FutureFetcher {
```

Comment:
Related to a previous comment of mine..here we are fetching baker rolls. `bakersFetcher` has ambiguous meaning. Could we rename this to something like `bakerRollsFetcher`?